privacyidea/api/before_after.py

```diff
@@ -247,7 +247,21 @@ def after_request(response):
 @postrequest(sign_response, request=request)
 def auth_error(error):
     if "audit_object" in g:
-        g.audit_object.log({"info": error.message})
+        message = ''
+        description = ''
```

Comment:
I think the ``description`` variable is still unused, could you remove it?

Reply:
yup